src/features/auth/pages/signup/SignupForm.native.test.tsx

```diff
@@ -227,71 +231,78 @@ describe('Signup Form', () => {
     expect(screen.getByText('Confirme ton adresse e-mail')).toBeOnTheScreen()
   })
 
-  it('should call logContinueSetEmail when clicking on next step from SetEmail', async () => {
-    renderSignupForm()
-
-    const emailInput = screen.getByPlaceholderText('tonadresse@email.com')
-    fireEvent.changeText(emailInput, 'email@gmail.com')
-    await act(() => fireEvent.press(screen.getByTestId('Continuer vers l’étape Mot de passe')))
-
-    expect(analytics.logContinueSetEmail).toHaveBeenCalledTimes(1)
-  })
+  describe('analytics', () => {
+    it('should trigger StepperDisplayed tracker when clicking on next step from SetEmail', async () => {
+      renderSignupForm()
 
-  it('should call logContinueSetPassword when clicking on next step from SetPassword', async () => {
-    renderSignupForm()
+      const emailInput = screen.getByPlaceholderText('tonadresse@email.com')
+      fireEvent.changeText(emailInput, 'email@gmail.com')
+      await act(() => fireEvent.press(screen.getByTestId('Continuer vers l’étape Mot de passe')))
 
-    const emailInput = screen.getByPlaceholderText('tonadresse@email.com')
-    fireEvent.changeText(emailInput, 'email@gmail.com')
-    await act(async () => {
-      fireEvent.press(screen.getByTestId('Continuer vers l’étape Mot de passe'))
+      expect(analytics.logStepperDisplayed).toHaveBeenNthCalledWith(
+        1,
+        StepperOrigin.HOME,
+        PreValidationSignupStep.Email
+      )
     })
 
-    const passwordInput = screen.getByPlaceholderText('Ton mot de passe')
-    await act(async () => fireEvent.changeText(passwordInput, 'user@AZERTY123'))
-    await act(async () =>
-      fireEvent.press(screen.getByTestId('Continuer vers l’étape Date de naissance'))
-    )
+    it('should trigger StepperDisplayed tracker when clicking on next step from SetPassword', async () => {
```

Comment:
A renommer, par exemple (t'as peut-être une meilleure proposition) : 
```suggestion
    it('should trigger StepperDisplayed tracker when displaying step', async () => {
```